### PR TITLE
feat(TMRX-1250): Updates to update button

### PR DESCRIPTION
**Pull request by bibicollins1**
Description

Minor updates to reset state after delay has passed so that it should catch the next update and render the button again. 
Having a problem with caching so have passed true into reload() to hopefully break that.

### CODE REVIEW

File: packages/article-skeleton/src/article-skeleton.js [hunk 1]
```diff
@@ -82,6 +82,8 @@ const ArticleSkeleton = ({
     isCommentEnabled
   } = article;
 
+  console.log(article.publishedTime, 'PUBLISHED TIME');
```
Comment (ibrahim-kurhan) [inline packages/article-skeleton/src/article-skeleton.js:85]: console.log?

**Review by adamosborne-tnl — APPROVED** [review]

File: packages/article-skeleton/src/article-skeleton.js [hunk 2]
```diff
@@ -260,7 +260,7 @@ const ArticleSkeleton = ({
                         label="New update"
                         handleClick={() => scrollToTopAndRefresh(window)}
                         arrowUp
-                        updatedTime={article.updatedTime}
+                        updatedTime={article.publishedTime}
```
Comment (ibrahim-kurhan) [inline packages/article-skeleton/src/article-skeleton.js:263]: just out of curiosity why are you using publishedTime instead of updatedTime?

**Review by ibrahim-kurhan — APPROVED** [review]

File: packages/article-skeleton/src/article-skeleton.js (hunk 2 above)
Comment (bibicollins1) [reply packages/article-skeleton/src/article-skeleton.js:263]: So turns out the updateTime changes automatically at any update, e.g. fixing a typo. Editorial has more control over publishedTime so they can change that when they make meaningful updates instead.

Comment (bibicollins1) [reply packages/article-skeleton/src/article-skeleton.js:263]: Apps currently have the feature pointing to publishedTime

**Pull request merged**